vue/src/modules/Auth/Register.vue

```diff
@@ -3,46 +3,71 @@
     <div class="col-md-4">
       <div class="login-left">
         <img src="/assets/img/apollo11-white.png" alt="" style="width: 80px"/>
-        <h3>Welcome</h3>
+        <h3>{{ $t('Welcome') }}</h3>
         <p>You are 30 seconds away from entering in <b>Agora!</b></p>
         <router-link class="btn btn-light btn-secondary btn-block" :to="{name: 'auth.login'}">Login</router-link>
       </div>
     </div>
     <div class="col-md-8 col-right">
       <div class="login-right clearfix">
-        <h3 class="login-heading">Create an account</h3>
+        <div v-if="loading" class="content-spinner text-center text-info">
+          <b-spinner class="align-middle"></b-spinner>
+          <strong>{{ $t('Please wait...') }}</strong>
+        </div>
+        <h3 class="login-heading">{{ $t('Create an account') }}</h3>
         <br>
         <div class="login-form">
```

Comment:
@skavt When copy/paste-ing the templates, please have double look on the classes and IDs, here it should be `register-form` instead of `login-form`. Also, if you do not use them in scss or any js logic, you can remove them.